appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecService.java

```diff
@@ -221,9 +221,10 @@ private void cancelScheduledScan() {
      *            the listener
      * @return a registration object that can be used to remove the listener
      */
-    public Registration addScanEventListener(AppSecScanEventListener listener) {
+    public Registration addScanEventListener(AppSecScanEventListener listener,
+            String listenerName) {
```

Comment:
This new parameter has the only purpose to feed a log message, feels a bit an overkill and it's also a breaking change.

If we want to identify the caller let's use the caller's logger instead, i.e. log the message right after adding the listener.

Reply:
That makes sense. Removed the parameter and added log messages in the caller class.